csharp/extractor/Semmle.Extraction.CIL/Entities/Method.cs

```diff
@@ -37,17 +37,15 @@ protected Method(IGenericContext gc) : base(gc.Context)
         public virtual IList<LocalVariable>? LocalVariables => throw new NotImplementedException();
         public IList<Parameter>? Parameters { get; protected set; }
 
-        public override void WriteId(TextWriter trapFile) => WriteMethodId(trapFile, DeclaringType, NameLabel);
-
         public abstract string NameLabel { get; }
 
-        protected internal void WriteMethodId(TextWriter trapFile, Type parent, string methodName)
```

Comment:
This is strange. Why was this `protected internal`?!

Reply:
No idea :-)